products/iap/api.yaml

```diff
@@ -61,6 +61,42 @@ objects:
         name: 'appId'
         description: Id of the App Engine application.
         required: true
+  - !ruby/object:Api::Resource
```

Comment:
Can you comment above each resource that they're fake resources to act as parents for IAM policies?